packages/server/test/security/api.test.ts

```diff
@@ -49,32 +50,58 @@ const shouldFail = async (func: Function, onError: (err: AxiosError) => void) =>
 }
 
 describe('API', () => {
-  const sync = async (data?: SyncRequest, config?: AxiosRequestConfig) => {
-    const client = http()
+  describe('Admin', () => {
+    const createClient = async () => {
+      const client = http()
+      const res = await client.post('/api/admin/clients')
 
-    const res = await client.post<SyncResult>('/api/sync', data, config)
+      expect(res.data).toEqual({ id: expect.anything(), token: expect.anything() })
+      expect(res.status).toEqual(201)
 
-    expect(res.data).toEqual({ id: expect.anything(), token: expect.anything(), webhooks: expect.anything() })
-    expect(res.status).toEqual(200)
+      return res.data
+    }
 
-    return res.data
-  }
+    test('Should be able to create clients', async () => {
+      const res = await createClient()
+
+      clients.first.clientId = res.id
+      clients.first.clientToken = res.token
+
+      {
+        const res = await createClient()
+
+        clients.second.clientId = res.id
+        clients.second.clientToken = res.token
+
+        expect(res.id).not.toEqual(clients.first.clientId)
+        expect(res.token).not.toEqual(clients.first.clientToken)
+      }
+    })
+  })
 
   describe('Sync', () => {
+    const sync = async (clientId?: string, clientToken?: string, data?: any, config?: AxiosRequestConfig) => {
+      const client = http(clientId, clientToken)
+
+      const res = await client.post<SyncResult>('/api/sync', data || {}, config)
+
+      expect(res.data).toEqual({ webhooks: expect.anything() })
+      expect(res.status).toEqual(200)
+
+      return res.data
+    }
+
     test('Should not allow any other methods than POST and OPTIONS', async () => {
```

Comment:
I think we can remove this test as the response is now 404 when you query this endpoint with an unknown method